### PR TITLE
Added table expand feature to display docker ID and labels.

### PR DESCRIPTION
**Pull request by Naveenkumar27**
Added table-row expansion feature. It expands the row of a table on clicking the "View Details" button to show the details of an instance such as docker Id and labels. The feature is implemented using Angular-Material table expand-row feature.

### CODE REVIEW

File: client/src/app/dashboard/table-all/table-all.component.ts [hunk 1]
```diff
@@ -44,9 +52,12 @@ export class TableAllComponent implements OnInit {
     @Input() compType: string;
 
 
-    displayedColumns = ['ID', 'name', 'host', 'portNumber', 'instanceState', 'action'];
+    displayedColumns = ['ID', 'name', 'host', 'portNumber', 'instanceState', 'Details', 'action'];
+    columnsToDisplay: string[] = ['dockerId', 'labels'];
     dataSource: MatTableDataSource<Instance> = new MatTableDataSource<Instance>(this.dataArray);
+    data = new MatTableDataSource<Instance>();
     dialogResult: string;
+    expandedElement: any;
```
Comment (janniclas) [inline client/src/app/dashboard/table-all/table-all.component.ts]: please don't use any data type

Comment (Naveenkumar27) [reply client/src/app/dashboard/table-all/table-all.component.ts]: "any" datatype is changed.

File: client/src/app/dashboard/table-all/table-all.component.ts [hunk 2]
```diff
@@ -171,6 +182,24 @@ export class TableAllComponent implements OnInit {
         });
     }
 
+     /**
+   * Function used to expand table row and show details of Docker and Labels
+   */
+  onRowClicked(row: Instance) {
+    let filteredList;
```
Comment (janniclas) [inline client/src/app/dashboard/table-all/table-all.component.ts]: please add a return type to the method and a type to the filteredList variable

File: client/src/app/dashboard/table-all/table-all.component.ts [hunk 3]
```diff
@@ -44,9 +52,12 @@ export class TableAllComponent implements OnInit {
     @Input() compType: string;
 
 
-    displayedColumns = ['ID', 'name', 'host', 'portNumber', 'instanceState', 'action'];
+    displayedColumns = ['ID', 'name', 'host', 'portNumber', 'instanceState', 'Details', 'action'];
+    columnsToDisplay: string[] = ['dockerId', 'labels'];
     dataSource: MatTableDataSource<Instance> = new MatTableDataSource<Instance>(this.dataArray);
+    data = new MatTableDataSource<Instance>();
     dialogResult: string;
+    expandedElement: number;
```
Comment (janniclas) [inline client/src/app/dashboard/table-all/table-all.component.ts]: the type of the expandedElement is not number but actually the table element, thus an instance.

**Pull request merged**